test_scripts/Smoke/Resumption/ATF_Resumption_BACKGROUND_level.lua

```diff
@@ -0,0 +1,177 @@
+--  Requirement summary:
+--  [HMILevel Resumption]: Both LIMITED and FULL applications must be included to resumption list
+--  Applications of BACKGROUND are not the case of HMILevel resumption in the next ignition cycle.
+--
+--  Description:
+--  Check that SDL performs app's data resumption and does not resume BACKGROUND HMI level  
+--  of media after transport unexpected disconnect on mobile side.
+
+--  1.  Used precondition
+--  App in  BACKGROUND
+--  Default HMI level is NONE.
+--  App has added 1 sub menu, 1 command and 1 choice set.
+
+--  2. Performed steps
+--  Turn off transport.
+--  Turn on transport.
+--
+--  Expected behavior:
+--  1. App is unregistered from HMI.
+--     App is registered on HMI, SDL resumes all data and App gets default HMI level NONE.
+
+-- [[ Required Shared Libraries ]]
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonStepsResumption = require('test_scripts/Smoke/Resumption/commonStepsResumption')
+local mobile_session = require('mobile_session')
+
+--[[ General Settings for configuration ]]
+Test = require('user_modules/dummy_connecttest')
+require('cardinalities')
+require('user_modules/AppTypes')
+
+-- [[Local variables]]
+local default_app_params1 = config.application1.registerAppInterfaceParams
+local default_app_params2 = config.application2.registerAppInterfaceParams
+local grammarIDValue = nil
```

Comment:
@istoimenova now it's used in data validation (line 165)
cf1069a1b7056bceafdc4d0951f5663f5362d4ee